index.rst

```diff
@@ -10,20 +10,849 @@
+This is especially important if connections are required to the summit instance.
+
+Lastly, the results of this analysis do not need to be forwarded back to the summit control system.
+
+Potential Paths for Implementation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Rapid Analysis payload relies heavily on SFP, and therefore is a compatible payload with both the DRP and the Alert Production Pipelines.
+The ultimate implementation decisions are outside the FAFF scope, however, because of the speed requirements, which will necessitate the pre-loading of expected image properties into memory (e.g. catalogues), it is expected that the path of least resistance would be to work with the Alert Production team in the development of Rapid Analysis.
+
+Another aspect which may impact implementation is that Rapid Analysis only needs to run once per frame.
+Even upon a failure to produce one of the parameters, or the publishing of an incorrect result, the system will not be rerun and therefore the database containing the results does not need to support versioning or relationships to previous results.
+
+A re-occurring concern has been whether or not the Antu servers can support the Rapid Analysis framework.
+FAFF has worked with Rubin project members to create a preliminary analysis of the compute required to run Rapid Analysis and found the following:
+
+.. at with ~250 cores (1 per detector and a handful for overhead), combined with some attention paid to code performance enhancements, we expect that in terms of processing, keeping up with a 30s image cadence is very feasible.
+
+
+- ~4 cores per CCD are required to perform the data processing
+- Using the full 189 CCDs also requires 756 cores which is nearly the entire Antu capability (784 cores)
+- To support required data Input/Output (I/O), a cluster would ideally have a small number of cores per node, then spread the data out across multiple disks.
+  Antu has a high core-to-node ratio, and is therefore likely unable to run Rapid Analysis for the entire array at a ~30s cadence.
+
+
+At the moment, it is unclear if the computing infrastructure could be augmented to support full-frame on-the-fly processing in Chile.
+If not, then the remaining option is to reduce the number of CCDs that get processed.
+DECam encountered the same constraints and invoked a pipeline that supports different configurations that specify various patterns of sensors to reduce.
+For example, pointing tests used just the central portion of the focal plane.
+A list of possible focal plane configurations should be created; we have already reached out to the AOS[#]_ and Science Verification[#]_ groups for suggestions.
+It is recommended that Rubin adopt a similar architecture as it is not expected that any summit-based Rapid Analysis image quality metrics would require the full array.
+Especially since the camera diagnostic cluster handles the low-level health checks for all sensors, as is discussed in `Deliverable 5`_.
```

Comment:
It feels like this phrase belongs to the one on top. Maybe merge them?

Reply:
I incorporated Angelo's comment and then re-worded this a bit.